packages/devtools_app/lib/src/shared/config_specific/import_export/import_export.dart

```diff
@@ -146,29 +145,17 @@ abstract class ExportController {
     bool notify = true,
   }) {
     fileName ??= ExportController.generateFileName(type: type);
-
-    switch (content) {
-      case String:
-        saveFile(content: content as String, fileName: fileName);
-      case Uint8List:
-        saveDataFile(content: content as Uint8List, fileName: fileName);
-      default:
-        throw 'Unsupported content type: $T';
-    }
-
+    saveFile(
+      content: content as String,
```

Comment:
should this cast be removed here since content could be String or Uint8List?

also, add the type to saveFile: `saveFile<T>(...)`